jack_mix_box.c

```diff
@@ -45,7 +45,7 @@ usage()
 
 	printf("Usage:\n");
 	printf("\tjack_mix_box [ -n|--name <jack client name> ] [ -s|--stereo ] [ -p|--pickup ] [ -v|--volume <initial vol> ] MIDI_CC_1 MIDI_CC_2 ...\n");
-	printf("\tsend SIGUSR1 to the process to have the current columes reported per input channel\n\n");
+	printf("\tsend SIGUSR1 to the process to have the current volumes reported per input channel\n\n");
```

Comment:
I reworked the `jack_mix_box` usage message in a commit to the main branch. Please rebase your branch and change the usage message to include a description of the `-p` option.

E.g.:

```diff
diff --git a/jack_mix_box.c b/jack_mix_box.c
index 7a45941..ca5d474 100644
--- a/jack_mix_box.c
+++ b/jack_mix_box.c
@@ -43,10 +43,11 @@ void
 usage()
 {
        printf("Usage: ");
-       printf("jack_mix_box [-n <name>] [-s] [-v <dB>] MIDI_CC...\n");
+       printf("jack_mix_box [-n <name>] [-p] [-s] [-v <dB>] MIDI_CC...\n");
        printf("\n");
        printf("-h|--help\tprint this help message\n");
        printf("-n|--name\tset JACK client name\n");
+       printf("-p|--pickup\tenable MIDI pickup mode (default: jump-to-value)\n");
        printf("-s|--stereo\tmake all input channels stereo with left+right input\n");
        printf("-v|--volume\tinitial volume gain in dB, default is 0.0 (i.e. unity gain)\n");
        printf("\n");
```

Reply:
ok